app/views/categories/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<section class='section cats-sect'>
+  <h1 class='title is-size-3 has-text-centered mt-5 cat_head'>
+   Categories
+  </h1>
+  <div class='container cat-sect'>
+    <% @categories.each do |category| %>
+      <div class='card'>
+        <div class='card-content'>
+          <div class='media'>
+            <div class='media-content'>
+              <div class='title-icon mb-5'>
+                <img src="<%= category.icon %>" alt="<%= category.name %>" class="image is-96x96">
+                <h2 class='subtitle is-5 cat_head'>
+                  <%= link_to category.name, category_category_expenditures_path(category) %>
+                </h2>
+                <p>Amount: $<%= category.total_expenses %></p>
+              </div>
+              <div class='actions mt-5'>
+                <span class='subtitle is-5'>
+                <%# link to edit %>
+                <%= link_to 'EDIT', edit_category_path(category), class: 'button is-primary is-small' %>
+                </span>          
+                <span class='subtitle is-5'>
+                  <%= link_to 'REMOVE', category_path(category.id), data: { turbo_method: :delete, turbo_confirm: "Are you sure?" }, class: 'button is-danger is-small' %>
+                </span>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    <% end %>
+  </div>
```

Comment:
- [ ] Your app working fine but you should stick to the design as required font, colors, layout and space elements as in the requirements 
- Screenshot of the requirements 
<img width="522" alt="Screenshot 2022-11-17 at 13 13 06" src="https://user-images.githubusercontent.com/61790587/202443960-d4cd1cc0-74fe-44fd-bd6e-b25dec798ba5.png">
- Yours 
<img width="386" alt="Screenshot 2022-11-17 at 13 46 02" src="https://user-images.githubusercontent.com/61790587/202450456-e7e7000b-0334-4fe4-9f23-31e7202182d6.png">
- Design 
<img width="270" alt="Screenshot 2022-11-17 at 13 46 17" src="https://user-images.githubusercontent.com/61790587/202450480-bfdb8c5b-20e7-48d0-b0ff-2c5819624123.png">

Reply:
I appreciate what your suggestion. But I like my design better and in any case, I will change it back after approval. I put a lot of work into creating my table, etc. I don't want to have to change it. So, if possible, I would like to request that you forgo this requirement, at least for this page because my app works and I like it and I don't want to change everything just to show that I can style a page using CSS. Thank you.